server/settings.js

```diff
@@ -428,9 +428,26 @@ var updateSettings = function (updateQuery, callback) {
 exports.initialize = function(callback){
     getAllSettings(function(err, allSettingsObj){
         if(err){
```

Comment:
I already made the changes to cover for that

Reply:
Why are you doing it again

Reply:
I knew I did that before, someone dropped my work :'( anyway you don't need all of this. In db.js when it fails to find the settings collection just force it to create one. I sent you the code offline

Reply:
done